app/Http/Controllers/Api/LicensesController.php

```diff
@@ -12,6 +12,10 @@
 use App\Models\LicenseSeat;
 use Illuminate\Http\Request;
 use Illuminate\Support\Facades\DB;
+use App\Http\Controllers\Api\User;
```

Comment:
I'm not sure why we'd be using a controller here?

Reply:
Dear Snipe,
on the the licensescontroller, as it uses user_id to refer to wich user the license is going to be checked on checkout funtion it complained when trying that this App\Http\Controllers\Api\User controller was not present, the log on laravel.log after setting the line the error dissapeared and the checkout worked.